validator/main.py

```diff
@@ -59,36 +60,45 @@ class RestrictToTopic(Validator):
         disable_classifier (bool, Optional, defaults to False): controls whether
             to use the Zero-Shot model. At least one of disable_classifier and
             disable_llm must be False.
+        classifier_api_endpoint (str, Optional, defaults to None): An API endpoint
+            to recieve post requests that will be used when provided. If not provided, a
+            local model will be initialized.
         disable_llm (bool, Optional, defaults to False): controls whether to use
             the LLM fallback. At least one of disable_classifier and
             disable_llm must be False.
-        model_threshold (float, Optional, defaults to 0.5): The threshold used to
+        zero_shot_threshold (float, Optional, defaults to 0.5): The threshold used to
             determine whether to accept a topic from the Zero-Shot model. Must be
             a number between 0 and 1.
+        llm_threshold (int, Optional, defaults to 3): The threshold used to determine
+        if a topic exists based on the provided llm api. Must be between 0 and 5.
     """
```

Comment:
I think Args should be updated now that device not only takes int

Reply:
device is a Optional[Union[str, int]] now to support cpu/mps or an int for device

Reply:
Exactly. What I meant is here in the docstrings it still appear as 
```
device (int, Optional, defaults to -1): Device ordinal for CPU/GPU
supports for Zero-Shot classifier. Setting this to -1 will leverage
CPU, a positive will run the Zero-Shot model on the associated CUDA
device id.
```

Reply:
Oh yeah good catch. Sorry, I missed that. Fixed